backend/migrate_encrypt_tokens.py

```
@@ -0,0 +1,115 @@
+            # Check if token is already encrypted
+            if is_token_encrypted(git_token):
+                logger.info(f"User {user.user_name} token already encrypted, skipping")
+                continue
+
+            # Encrypt the token
+            try:
+                encrypted_token = encrypt_git_token(git_token)
+                git_item["git_token"] = encrypted_token
+                modified = True
+                logger.info(f"Encrypted token for user {user.user_name}, type: {git_item.get('type')}")
+            except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)